kernels/kernel.cl

```diff
@@ -6,8 +6,9 @@
 # define NATIVE_LITTLE_ENDIAN 1
 #endif
 
-#define BLAKE2S_BLOCKBYTES 64
+#define COMPARE_ALL
```

Comment:
What's this mean, again

Reply:
See: https://github.com/kadena-community/bigolchungus/blob/c4a55f8824150daeb4f82d11e72285a5c151f575/kernels/kernel.cl#L269

For some reason, running all of the comparisons gave me 0.5% faster hashing speed (955 -> 960 MH/s). We assume it has something to do with cache hits or something?

Reply:
Lmao